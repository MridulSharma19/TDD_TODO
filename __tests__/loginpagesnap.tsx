import LoginPage from "@/components/loginpage/loginpage";
import { render } from "@testing-library/react";

it("renders loginpage unchanged", () => {
    const dummyFunc = jest.fn();

    const { container } = render(<LoginPage setName={dummyFunc} />);
    expect(container).toMatchSnapshot();
});
