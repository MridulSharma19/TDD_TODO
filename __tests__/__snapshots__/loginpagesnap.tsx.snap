// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loginpage unchanged 1`] = `
<div>
  <div
    class="home-container login-page"
  >
    <h1>
      TODO APP
    </h1>
    <h2>
      Please enter your name
    </h2>
    <form
      class="form-group"
    >
      <input
        id="name"
        name="name"
        placeholder="Your name"
        type="text"
        value=""
      />
      <input
        hidden=""
        type="submit"
      />
    </form>
    <button
      disabled=""
    >
      Get Started
    </button>
  </div>
</div>
`;
